mlflow/tensorflow.py

```diff
@@ -1,4 +1,17 @@
-"""MLflow integration for Tensorflow."""
+"""MLflow integration for Tensorflow.
+
+Manages logging and loading Tensorflow models as Python Functions. You are expected to save your own
+``saved_models`` and pass their paths to ``log_saved_model()`` 
+so that MLflow can track the models. 
+
+In order to load the model to predict on it again, you can call
+``estimator = mlflow.pyfunc.load_pyfunc(saved_model_dir)``, followed by 
+``prediction= estimator.predict(pandas DataFrame)`` in order to obtain a prediction in a pandas DataFrame.
+
+Note that loading the model back via ``load_pyfunc()`` does not load the actual model, but rather
```

Comment:
The wording here is a bit confusing since we do load back the saved model (interestingly TF saved models don't have to come from an estimator). Maybe we can say something like "Note that the loaded PyFunc model does not expose any APIs for model training"?